build/grunt.js

```diff
@@ -31,7 +32,7 @@ module.exports = function(grunt) {
       release: {
         tag_name: 'v'+ version.full,
         name: version.full,
-        body: chg.find(version.full).changesRaw
+        body: nodeCli.exec('conventional-changelog', '-p videojs', {silent: true}).output
```

Comment:
im not sure its worth using `shelljs-nodecli` for this, shouldn't we just use `shelljs` and point to `path.join(__dirname, '..', 'node_modules', '.bin', 'conventional-changelog')`. My reasoning here is that `shelljs-nodecli` is not very heavily used, spin off modules are usually not kept up to date with the parent, and I don't even see a git repo for it on npm.